### DISCUSSION TITLE
ScriptReference/Graphics.RenderMeshInstanced

### DISCUSSION THREAD

**Discussion by giscus** [Developer-Notes-Extension/Unity-Notes#656] [Notes]
# ScriptReference/Graphics.RenderMeshInstanced

https://docs.unity3d.com/ScriptReference/Graphics.RenderMeshInstanced.html

**Comment by redwyre** [comment]
Note that you should prefer to create the `RenderParams` using the constructor that takes the material as this will set sensible defaults, otherwise you will need to manually set `renderingLayerMask` to a valid value or nothing will get rendered.
